crates/hir-def/src/item_tree.rs

```diff
@@ -753,7 +753,8 @@ bitflags::bitflags! {
         const HAS_CONST_KW = 1 << 3;
         const HAS_ASYNC_KW = 1 << 4;
         const HAS_UNSAFE_KW = 1 << 5;
-        const IS_VARARGS = 1 << 6;
+        const HAS_SAFE_KW = 1 << 6;
+        const IS_VARARGS = 1 << 7;
```

Comment:
Is there is a special reason you inverted those?

Reply:
No particular reason, I just want it to be after `unsafe_kw`. It is better to append at the end indeed.